OpenOversight/app/models/database.py

```diff
@@ -706,7 +818,9 @@ class Incident(BaseModel, TrackUpdates):
         db.Integer, db.ForeignKey("locations.id", name="incidents_address_id_fkey")
     )
     address = db.relationship(
-        "Location", backref=db.backref("incidents", cascade_backrefs=False)
+        "Location",
+        backref=db.backref("incidents", cascade_backrefs=False),
+        lazy="joined",
```

Comment:
Addresses this periodic error: `sqlalchemy.orm.exc.DetachedInstanceError: Parent instance <Incident at 0xffff731ba510> is not bound to a Session; lazy load operation of attribute 'address' cannot proceed (Background on this error at: h...`